third_party/versions.bzl

```diff
@@ -11,6 +11,14 @@ Note that each of the dependencies will have all ':'s & '.'s replaced with under
 the dependency "android.arch.core:core-testing": "1.1.1" will be referencable with the following:
 //third_party:android_arch_core_core-testing (no version is included so that versions can be easily
 updated here).
+
+Note that after making any change in 'MAVEN_PRODUCTION_DEPENDENCY_VERSIONS' or
+'MAVEN_TEST_DEPENDENCY_VERSIONS' dicts (e.g - Updating version of some dependencies, adding/deleting
+any dependencies) you'll need to run the command: REPIN=1 bazel run @unpinned_maven//:pin to update
```

Comment:
Should this command be in a separate line? It might be difficult for some people of understand what exactly is command in this entire line.

Reply:
Maybe just rely on the link below, instead, since it's easier to keep that up-to-date than code.

Reply:
Done.